source/ACCESS.md

```diff
@@ -35,6 +36,18 @@ datalad install -r git@github.com:courtois-neuromod/cneuromod.git
 cd cneuromod
 ```
 
+### DTA restricted data.
+
+Some participants have not asked CNeuroMod and consented for a fully public release of their data through CONP and these are only accessible through the DTA. The s3 access_key and secret_key will be provided by the data manager after being granted access to cneuromod by the user access committee.
```

Comment:
Maybe this is clearer? Data from two of the six participants are restricted ( i.e not publically available CONP) and require researcher to apply for [access](https://www.cneuromod.ca/access/access/l). Upon approval by the CNeuroMod User Access Committee, and once the DTA is signed by institutional representatives, the data manager will provide the S3 access_key and secret_key.